src/OSPSuite.Core/Domain/PKAnalyses/UserDefinedPKParameter.cs

```diff
@@ -5,16 +5,22 @@ namespace OSPSuite.Core.Domain.PKAnalyses
    public enum StandardPKParameter
    {
       Unknown,
-      Cmax,
-      Cmin,
-      Tmax,
-      Tmin,
-      CTrough,
-      AucTend,
-      AucmTend,
-      AucInf,
-      AucTendInf,
-      Mrt,
+      C_max,
+      C_max_norm,
```

Comment:
I have added all the norm parameters.This should allow an automatic calculation of those parameters from R using tD1 and tD2